### DISCUSSION TITLE
Chia blockchain Peak Time is not being sync with my desktop

### DISCUSSION THREAD

**Discussion by sftaa** [Chia-Network/chia-blockchain#8047] [Support]
Dear Team,

Chia blockchain Peak Time is not being sync with my desktop, please assist how this issue can fix. Due to this my Synchronization is stuck. Thank you in Advance. 

BR

**Comment by BrandtH22** [comment]
@sftaa , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the desktop time settings should have the ability to synchronize the time, if not then you will need to manually set the time on the machine. I like to use this site to confirm whether my system is in sync https://time.is/

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
